src/main/java/mate/controller/GetAllDriversController.java

```diff
@@ -0,0 +1,31 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/all")
+public class GetAllDriversController extends HttpServlet {
+    private DriverService driverService;
+
+    @Override
+    public void init() throws ServletException {
+        Injector injector = Injector.getInstance("mate");
+        driverService = (DriverService) injector.getInstance(DriverService.class);
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Driver> driversList = driverService.getAll();
+        req.setAttribute("drivers", driversList);
```

Comment:
```suggestion
        req.setAttribute("drivers", driverService.getAll());
```
lets make your code consistent